docs/running.rst

```diff
@@ -14,10 +14,23 @@ the site.
 
 The recommended ways to run Wooey are:
 
+Through two separate processes
+------------------------------
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
```

Comment:
'on windows, the --best option may not be supported' sounds like it could never be supported, which is contrary to the previous comment you had on this line. What do you think?